### PR TITLE
Seanmcm/0 20 1 release

### PR DESCRIPTION
**Pull request by sean-mcmanus**

### CODE REVIEW

File: launch.md [hunk 1]
```diff
@@ -22,7 +22,9 @@ Set or change the following options to control VS Code's behavior during debuggi
   Tells _GDB or LLDB_ what paths to search for .so files. Separate multiple paths with a semicolon. Example: `"/Users/user/dir1;/Users/user/dir2"`.
 
 * #### `externalConsole`
-  If set to `true`, launches an external console for the application. If `false`, no console is launched and VS Code's debugging console is used. Note this option is ignored in some cases for technical reasons.
+  Windows: When set to true, it will spawn an external console. When set to false, it will use VS Code's integratedTerminal.
+  Linux: When set to true, it will notify VS Code to spawn an external console. When set to false, it will use VS Code's integratedTerminal
```
Comment (sean-mcmanus) [inline launch.md]: This line is missing a period. I fixed it in https://github.com/Microsoft/vscode-cpptools/pull/2750, along with other minor issues.

File: Extension/package.json [hunk 2]
```diff
@@ -1411,14 +1411,14 @@
     "vscode-cpptools": "2.1.1",
     "vscode-debugadapter": "~1.24.0",
     "vscode-debugprotocol": "~1.24.0",
-    "vscode-extension-telemetry": "~0.1.0",
+    "vscode-extension-telemetry": "~0.0.22",
```
Comment (pieandcakes) [inline Extension/package.json:1414]: did you mean to roll this back?

**Review by pieandcakes — APPROVED** [review]

File: Code Samples/Fib/.vscode/tasks.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "version": "2.0.0",
```
Comment (pieandcakes) [inline Code Samples/Fib/.vscode/tasks.json:4]: did you remove the colon? I Don't know what this diff means>?

Comment (sean-mcmanus) [reply Code Samples/Fib/.vscode/tasks.json:4]: No, there's no colon removed. Are you seeing a change on that line?

File: Extension/package.json (hunk 2 above)
Comment (sean-mcmanus) [reply Extension/package.json:1414]: Yes, they regressed our telemetry -- they recommended we roll back until it's fixed.

File: Code Samples/Fib/.vscode/tasks.json (hunk 3 above)
Comment (pieandcakes) [reply Code Samples/Fib/.vscode/tasks.json:4]: the colon is marked as "red" I don't know why

Comment (sean-mcmanus) [reply Code Samples/Fib/.vscode/tasks.json:4]: Yeah, I think that's just some GitHub visualization bug -- it's red in the pre-post versions. The raw data doesn't have that.

Comment (bobbrow) [reply Code Samples/Fib/.vscode/tasks.json:4]: It's red because the comments as messing up the formatter since comments are not valid in JSON.

Comment (sean-mcmanus) [reply Code Samples/Fib/.vscode/tasks.json:4]: Ah, that makes sense -- VS Code uses a json parser for tasks.json that allows comments though, so it's just a GitHub issue.

**Pull request merged**